alternator/executor.cc

```diff
@@ -7,7 +7,6 @@
  */
 
 #include "utils/base64.hh"
-
 #include <seastar/core/sleep.hh>
 #include "alternator/executor.hh"
```

Comment:
Such a complex change  ("alternator: add caching of expressions") should come with explanations.